src/components/views/rooms/MessageComposer.js

```diff
@@ -390,18 +388,6 @@ export default class MessageComposer extends React.Component {
             );
         }
 
-        let autoComplete;
```

Comment:
Doesn't this want to be taken out of the if block rather than removed altogether?